### PR TITLE
A lot of code calls l()/url(), let's wrap it, so we can mock it.

### PR DESCRIPTION
**Pull request by dawehner**

### CODE REVIEW

**Comment by fabianx-ai** [comment]
**RTBM**

File: lib/Drupal/service_container/Tests/LinkGeneratorTest.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      'description' => '',
+      'group' => 'service_container',
+    );
+  }
+
+  /**
+   * Adds some really basic integration test.
+   */
+  public function testL() {
+    $result = l('Title', 'test-path');
```
Comment (fabianx-ai) [inline lib/Drupal/service_container/Tests/LinkGeneratorTest.php:27]: I think this should be calling the LinkGenerator from the container and not l() directly, which is not test coverage we need.

File: lib/Drupal/service_container/Tests/UrlGeneratorTest.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      'description' => '',
+      'group' => 'service_container',
+    );
+  }
+
+  /**
+   * Adds some really basic integration test.
+   */
+  public function testUrl() {
+    $result = url('test-path');
```
Comment (fabianx-ai) [inline lib/Drupal/service_container/Tests/UrlGeneratorTest.php:27]: I think this should be calling the UrlGenerator from the container and not l() directly, which is not test coverage we need.

Comment (dawehner) [inline lib/Drupal/service_container/Tests/UrlGeneratorTest.php:27]: Yeah I wasn't concentrated at all here: https://github.com/LionsAd/service_container/commit/e950cc9393d3ce5bc89f734327298d265fcf4bd9

**Pull request merged**